netci.groovy

```diff
@@ -1813,14 +1813,13 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                                 Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Innerloop Build and Test")
                             }
                             break
-                        //no_tiered_compilation_innerloop will be added as default once it is confirmed working
-                        //case 'no_tiered_compilation_innerloop':
-                        //    // Default trigger
-                        //    if (configuration == 'Checked') {
-                        //        def displayStr = getStressModeDisplayName(scenario)
-                        //        Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Build and Test (Jit - ${displayStr})")
-                        //    }
-                        //    break
+                        case 'no_tiered_compilation_innerloop':
+                            // Default trigger
+                            if (configuration == 'Checked') {
+                                def displayStr = getStressModeDisplayName(scenario)
+                                Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Build and Test (Jit - ${displayStr})")
```

Comment:
IMO, this should include the word "Innerloop" like the "standard" innerloop job:
```
Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} Innerloop Build and Test (Jit - ${displayStr})")
```
Same for x86, below.